fixtures.go

```diff
@@ -189,6 +189,9 @@ var fixtures = Fixtures{{
 }, {
 	Tags:         []string{"packfile", "pack-sha256"},
 	PackfileHash: "407497645643e18a7ba56c6132603f167fe9c51c00361ee0c81d74a8f55d0ee2",
+}, {
+	Tags:         []string{"packfile", "Notes"},
```

Comment:
All the tags are lowercased, so it would be good to be keep it consistent: 
```suggestion
	Tags:         []string{"packfile", "notes"},
```

Reply:
Do I need to regenerate the tag in the git repo?

Reply:
Hey @JakobDev, no that is not required. The "Tags" here are handled based on the string values you provide on this array and is checked [here](https://github.com/go-git/go-git-fixtures/blob/master/fixtures.go#L222). It does not relate to any Git objects you may or may not have within your pack.

Reply:
Please rebase your changes.

Reply:
It is now lowercase